build-logic/basics/src/main/kotlin/gradlebuild.repositories.gradle.kts

```diff
@@ -19,4 +19,11 @@ repositories {
         name = "Gradle libs"
         url = uri("https://repo.gradle.org/gradle/libs")
     }
+    maven {
+        name = "CHAMP libs"
+        url = uri("https://releases.usethesource.io/maven/")
+        mavenContent {
+            includeModule("io.usethesource", "capsule")
```

Comment:
The parent pom (and modules) are also needed I guess?

Reply:
Thanks, @blindpirate, that was it.